### DISCUSSION TITLE
Adding column and row from table header

### DISCUSSION THREAD

**Discussion by mattratliff-innovations** [facebook/lexical#6798] [Q&A]
The editor generated from the main branch includes a table that has several header cells that provide additional features such as adding a column to the left or right, etc... I cannot find in the example or documentation how to implement this.  

![2024-11-05_16-51-10](https://github.com/user-attachments/assets/c312d18e-0e18-4916-a9ca-15c72662d984)

If anyone has a working code sample of this, I would be interested in knowing how you implemented it.  Thanks!

**Comment by etrepum** [comment]
This is the table action menu plugin in the playground

**Reply by mattratliff-innovations** [reply]
Great, thank you.
